app/javascript/src/hyacinth_v1/components/digital_objects/rights/rights_form/subsections/DescriptiveMetadata.js

```diff
@@ -84,14 +85,10 @@ function DescriptiveMetadata(props) {
           ))
         }
 
-        {
-          (descriptiveMetadata.title || [{}]).map((t, i) => (
-            <InputGroup key={i}>
-              <Label sm={4} align="right">Title</Label>
-              <ReadOnlyInput sm={8} value={t.sortPortion} />
-            </InputGroup>
-          ))
-        }
+        <InputGroup key="title_group">
+          <Label sm={4} align="right" for="rights_title_display">Title</Label>
+          <ReadOnlyInput sm={8} inputName="rights_title_display" value={(title || { value: {} }).value.sortPortion} />
```

Comment:
```suggestion
          <ReadOnlyInput sm={8} inputName="rights_title_display" value={title?.value?.sortPortion} />
```
I think this will work now that we have the optional chaining babel addition.

Reply:
Done!